libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-value.ts

```diff
@@ -1,13 +1,6 @@
+import { SkyFilterAdapterFilterValue } from '@skyux/lists';
+
 /**
  * Represents a value for a filter item.
  */
-export interface SkyFilterBarFilterValue {
-  /**
-   * The real value for the filter.
-   */
-  value: unknown;
-  /**
-   * A human-readable string for use with values that can't be displayed to the user.
-   */
-  displayValue?: string;
-}
+export type SkyFilterBarFilterValue = SkyFilterAdapterFilterValue;
```

Comment:
Changing a type from `interface` to `type` is technically a breaking change. 
See: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

If you need to share the same properties between the two interfaces, you can use `extends` instead.